src/main/jbake/content/blog/2017/06/23/opentracing-appmetrics.adoc

```diff
@@ -0,0 +1,270 @@
+= Using OpenTracing to collect Application Metrics in Kubernetes
+Gary Brown
+2017-6-23
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, tracing, jaeger, opentracing, kubernetes, prometheus
+
+This article will show how http://opentracing.io/[OpenTracing] instrumentation can be used to
+collect Application Metrics, in addition (but independent) from reported tracing data, from services
+deployed within https://kubernetes.io/[Kubernetes].
+
+== The example application
+
+In a recent article we showed how a Spring Boot application could easily be
+http://www.hawkular.org/blog/2017/06/9/opentracing-spring-boot.html[instrumented using OpenTracing].
+
+The https://github.com/objectiser/opentracing-prometheus-example[example] we are going to use in
+this article uses the same approach to create two services, _service1_ and _service2_ (sorry, not very
```

Comment:
Its not too late - I will probably need to rerun and get new images once the jaeger client is released, as currently the version is showing in the tag, which doesn't look nice. So can update.